deploy/handler/operator.yaml

```diff
@@ -56,38 +56,85 @@ spec:
               value: "False"
             - name: PROFILER_PORT
               value: "6060"
-            - name: CA_ROTATE_INTERVAL
-              value: {{ .CARotateInterval | default "8760h0m0s" }}
-            - name: CA_OVERLAP_INTERVAL
-              value: {{ .CAOverlapInterval | default "24h0m0s" }}
-            - name: CERT_ROTATE_INTERVAL
-              value: {{ .CertRotateInterval | default "4380h0m0s" }}
-            - name: CERT_OVERLAP_INTERVAL
-              value: {{ .CertOverlapInterval | default "24h0m0s" }}
           ports:
-          - containerPort: 8443
+          - containerPort: 9443
```

Comment:
will it have future value to put this in a env var (prevent future collisions)?

Reply:
we don't run it with network=host so is not going to collision, I doubt we are going to have multiple containers at the same pod with the same port.